test/Dialect/FIRRTL/xmr.mlir

```diff
@@ -0,0 +1,115 @@
+// RUN: circt-opt %s -split-input-file
+
```

Comment:
Some comment on these tests indicating that they are just expected to not error would be good.